src/Term/AliasGroup.php

```diff
@@ -19,7 +19,7 @@
 class AliasGroup implements Comparable, Countable {
 
 	/**
-	 * @var string
+	 * @var string Requested and actual language of the aliases.
```

Comment:
Ah, so you are putting the documentation of what the field means in the subclass here because that is where the field is defined? The issue with that is that you have the base class docs know about a subclass that it does not depend on. That is confusing for someone who is just interested in the base class. Because of this I think it's much better to document this in the derivative.